parsing/dialects/intermediate/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ValidatorTest.java

```diff
@@ -101,4 +101,32 @@
         + "FROM `GHI` AS `GHI`";
     sql(sql).rewritesTo(expected);
   }
+
+  @Test public void testCreateTableNoColumns() {
```

Comment:
Not sure of omitting the columns list altogether is legal. Instead, should we just do a test case with a table with no columns?